caf-audit-monkey-container/pom.xml

```diff
@@ -93,7 +93,7 @@
                             <name>${dockerCafAuditOrg}audit-monkey${dockerProjectVersion}</name>
                             <build>
                                 <maintainer>gregory.lucy@hpe.com</maintainer>
-                                <from>java:8</from>
+                                <from>cafapi/opensuse-jre8:1.2</from>
```

Comment:
Since this new base image already includes `tini` we should remove the unnecessary complexity of pulling it in again in the POM, like has been done in the other POM modified.

Reply:
Removed